extensions/spring-data-jpa/deployment/src/main/java/io/quarkus/spring/data/deployment/SpringDataJPAProcessor.java

```diff
@@ -66,6 +85,46 @@ void build(CombinedIndexBuildItem index,
                 interfacesExtendingCrudRepository, indexIndex);
     }
 
+    private void detectAndLogSpecificSpringPropertiesIfExist() {
+        Config config = ConfigProvider.getConfig();
+
+        Iterable<String> iterablePropertyNames = config.getPropertyNames();
+        List<String> propertyNames = new ArrayList<String>();
+        iterablePropertyNames.forEach(propertyNames::add);
+        List<String> springProperties = propertyNames.stream().filter(s -> pattern.matcher(s).matches()).collect(toList());
+        String notSupportedProperties = "";
```

Comment:
It might be better to be use a StringBuilder here, but it's not terribly important

Reply:
Yes, I hesitated